namerd/examples/k8s.yaml

```diff
@@ -0,0 +1,12 @@
+# This file assumes you are running through `kubectl proxy` on localhost:8001.
+storage:
+  kind: io.buoyant.namerd.storage.experimental.k8s
```

Comment:
I agree - I was going with the precedent from the other two storage `kind`s  here, which also use class paths.